### DISCUSSION TITLE
Is it possible to swipe back to a previous page on the first tab?

### DISCUSSION THREAD

**Discussion by wong2** [react-native-community/react-native-tab-view#1217] [Q&A]
Say I have two pages: `Home` and `Settings`, and I'm using `react-navigation` for routing, which enables gesture to swipe back to Home page from Settings page.
But after adding tab view to the Settings page, I can't use gesture to go back anymore!
Is it possible to swipe back to `Home` page if I'm on the first tab?

if there's any way to do this, I'd like to work on a PR, but please let me know how it's possible

**Comment by dancixx** [comment]
Any update on this topic?
